client/app/cloud/project/billing/consumption/estimate/cloud-project-billing-consumption-estimate.component.js

```diff
@@ -0,0 +1,15 @@
+import controller from './cloud-project-billing-consumption-estimate.controller';
+import template from './cloud-project-billing-consumption-estimate.html';
+
+const component = {
+  bindings: {
+    me: '<',
+  },
+  controller,
+  template,
+};
+
+export default component;
+
+angular.module('managerApp')
```

Comment:
Could be in another file, if we want to use this file without AngularJS then it will fail

Reply:
This module would contain files that I did not refactor to use `import/export`